bookshelf.js

```diff
@@ -0,0 +1,71 @@
+
+
+let bttn = document.createElement( 'button' );
+bttn.setAttribute( 'id', 'fetch-books-btn' );
+
+let text = document.createTextNode( 'Fetch Books' );
+bttn.appendChild( text );
+bttn.addEventListener( 'click', fetchBooks );
+
+document.body.appendChild( bttn );
+
+function fetchBooks() {
+    const booksJSON = 'https://raw.githubusercontent.com/codeyourfuture/bookshelf-project/master/books.json';
+    fetch( booksJSON )
+        .then( response => response.json() )
+        .then( processBooks )
+        .then( removeBtn )
+        .then( addArrows )
+}
+
+// processBooks function
+var list = document.createElement( 'ul' );
+
+var processBooks = booksJSON => {
+   
+   booksJSON.forEach( function( el, index ) {
+       var lists = document.createElement( 'li' );
+       lists.setAttribute( 'id', booksJSON[index].id );
+       lists.innerHTML = booksJSON[index].title + ' by ' + booksJSON[index].author;
+       list.appendChild( lists );
+        
+   });
+
+   document.body.appendChild( list );
+}
+
+/// Function to remove the button once the unordered list is created
+
+var removeBtn = () => {
+    var btnToRemove = document.getElementById( 'fetch-books-btn' );
+    btnToRemove.removeEventListener( 'click', fetchBooks );
+    btnToRemove.remove();
+}
+
+// Function to add up and down arrows
+function addArrows() {
```

Comment:
It's good that you separated this function out - it makes it cleaner to read. However I think you could make it a bit better by calling the function from within the `processBooks` function and passing the `list` as an argument. This would mean that you can create the `list` element within the `processBooks` function and not have it in a parent scope, which is a bit confusing to me.